src/injected/analyzers/tab-stops-requirement-result-processor.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { BaseStore } from 'common/base-store';
+import { VisualizationScanResultData } from 'common/types/store-data/visualization-scan-result-data';
+import { TabStopRequirementActionMessageCreator } from 'DetailsView/actions/tab-stop-requirement-action-message-creator';
+import { AllFrameRunner } from 'injected/all-frame-runner';
+import { AutomatedTabStopRequirementResult } from 'injected/tab-stop-requirement-result';
+import { isEqual } from 'lodash';
+
+export class TabStopsRequirementResultProcessor {
+    private seenTabStopRequirementResults: AutomatedTabStopRequirementResult[] = [];
+    private isStopped: boolean = true;
+
+    constructor(
+        private readonly tabStopRequirementRunner: AllFrameRunner<AutomatedTabStopRequirementResult>,
+        private readonly tabStopRequirementActionMessageCreator: TabStopRequirementActionMessageCreator,
+        private readonly visualizationResultsStore: BaseStore<VisualizationScanResultData>,
+    ) {}
+
+    public start = (): void => {
+        if (!this.isStopped) {
+            return;
+        }
+
+        this.seenTabStopRequirementResults = [];
+        this.tabStopRequirementRunner.topWindowCallback = this.processTabStopRequirementResults;
+        this.tabStopRequirementRunner.start();
+
+        this.isStopped = false;
+    };
+
+    public listenToStore(): void {
+        this.visualizationResultsStore.addChangedListener(this.onStateChange);
```

Comment:
The current implementation means we end up listening to all visualization store changes but don't think we need to do that.

I think we shouldn't be listening to the store unless the class has been _started_. I think moving the `addChangedListener` to `start` seems like a good idea (and similarly adding a `removeChangedListener` in `stop`).